Moskalenkov-homework-2.cpp

```diff
@@ -0,0 +1,153 @@
+#include <iostream>
+
+int main() {
+    std::cout << "What task do you want to solve ?" << std::endl;
+
+    int setNumber(0);
```

Comment:
настоятельно рекомендую использовать универсальную инициализацию
```suggestion
    int setNumber{};
```